dev/restart_all.sh

```diff
@@ -1,6 +1,29 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
you replace `sh` by `bash`, the unix standard is `sh`, not installed by default on Unix systems like FreeBSD